### DISCUSSION TITLE
Playwright workflow?

### DISCUSSION THREAD

**Discussion by hatton** [cawa-93/vite-electron-builder#886] [Q&A] [answered]
Currently, every time I change something in the main process, I need to run `vite build` before playwright is going to see that. 

Is that the workflow people use with vite/electron/playwright, or is there a more efficient way?

**Comment by makp0** [comment]
Hey @cawa-93, I'm also very curious about the workflow you've established with playwright.
After a short investigation, I can see that you've opted for an executable instead of ["."] 678188f1fb20763a6c726c4ff378b594cc4cc1e3. But this change didn't introduce the requirement of building the app before testing. 

Could you please explain why such decision was made with a tradeoff of tests taking way longer than if the source code was run directly in e2e. Your response is highly appreciated!

**Reply by cawa-93** [reply, marked as answer]
Cous e2e also should cover the compilation. In some cases compiled app may lose some node_modules and will not work if original `node_modules` dir doesnt exist
